### PR TITLE
Add check for ZOPEN_ROOTFS before refreshing zopen

### PR DESCRIPTION
**Pull request by IgorTodorovskiIBM**
To address the situation when someone downloads metaport and initializes a new filesystem with `zopen init`. We don't want to be refreshing something that does not exist.

### CODE REVIEW

File: buildenv [hunk 1]
```diff
@@ -138,7 +138,9 @@ zopen_install()
 
 zopen_append_to_setup() {
 cat <<ZZ
-  \$PWD/bin/zopen init --refresh
+  if [ -n "\$ZOPEN_ROOTFS" ] && [ -d "\$ZOPEN_ROOTFS" ]; then
+    \$PWD/bin/zopen init --refresh
```
Comment (MikeFultonDev) [inline buildenv:142]: do you want to verify that \$PWD/bin/zopen is executable first in case we move stuff around in the future and break this layout?

Comment (MikeFultonDev) [reply buildenv:142]: There are probably other places we have this assumption so ok if we don't do this one - but seems a bit dangerous without a check

Comment (IgorTodorovskiIBM) [reply buildenv:142]: Yeah, curl, git, and a few others :) Although we have checks in place for the return code there: https://github.com/ZOSOpenTools/gitport/blob/main/buildenv#L109

**Comment by IgorTodorovskiIBM** [comment]
Depends on https://github.com/ZOSOpenTools/meta/pull/602

**Review by MikeFultonDev — APPROVED** [review]
LGTM

**Pull request merged**